nbs/examples/ElectricityLoadForecasting.ipynb

```diff
@@ -0,0 +1,3625 @@
+{
```

Comment:
<p>Check values of <code>SeasonalNaive</code> with <code>df_train.iloc[len(df_train)-24:len(df_train)]</code></p><p> </p><p>The <code>SeasonalNaive</code> is replicating the values starting on 2001-12-30 09:00:00 when it should start on 2001-12-30 01:00:00 since the forecast starts on 2001-12-31 01:00:00. This can also be seen in the plots below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Thanks for reporting this. There is a bug in the `SeasonalNaive` model. I fixed it in #299, but I will open a specific PR.